astro/src/content/docs/apis/identity-providers/_identity-provider-login-method-request-parameter.mdx

```diff
@@ -10,7 +11,8 @@ import APIField from 'src/components/api/APIField.astro';
    * `UseVendorJavaScript` - When logging in allow the {props.idp_display_name} JavaScript library to determine the login method.
 
   {props.idp_type === 'Google' && <>
-    <span class="text-green-600">Since 1.44.0</span> <br/>
-    <strong>If you are using a version of FusionAuth older than 1.44.0</strong>, <code>UsePopup</code> won't work. <code>UseRedirect</code> will continue to work after this date. Please see [the 1.44.0 release notes](/docs/release-notes/#version-1-44-0) for more information. This [forum post](/community/forum/topic/2329/upcoming-google-identity-provider-changes) has more details on an available workaround and upgrade process.
+    <AvailableSince since="1.44.0" /><br/>
```

Comment:
Could this use the `Aside` instead to put this inside the block?

Reply:
Good call. Done.